fuzzers/frida_libpng/Makefile.toml

```diff
@@ -100,6 +100,14 @@ timeout 10s ./${FUZZER_NAME} -F LLVMFuzzerTestOneInput -H ./libpng-harness.so -l
 '''
 dependencies = [ "fuzzer", "harness" ]
 
+[tasks.test_windows]
+script_runner = "@shell"
+script='''
+start "" "./${FUZZER_NAME}" -F LLVMFuzzerTestOneInput -H ./harness_win.dll -l ./harness_win.dll --cores=0
+timeout /t 10 && taskkill /im ${FUZZER_NAME} /F
```

Comment:
this will probably never find out if it worked, right?
Maybe check if the out folder contains something, afterwards?